sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConsumer.cs

```diff
@@ -188,18 +188,24 @@ internal class AmqpConsumer : TransportConsumer
         ///   Receives a batch of <see cref="EventData" /> from the Event Hub partition.
         /// </summary>
         ///
-        /// <param name="maximumMessageCount">The maximum number of messages to receive in this batch.</param>
-        /// <param name="maximumWaitTime">The maximum amount of time to wait to build up the requested message count for the batch; if not specified, the per-try timeout specified by the retry policy will be used.</param>
+        /// <param name="maximumEventCount">The maximum number of messages to receive in this batch.</param>
+        /// <param name="maximumWaitTime">The maximum amount of time to wait for events to become available, if no events can be read from the prefetch queue. If not specified, the per-try timeout specified by the retry policy will be used.</param>
```

Comment:
Oh no, where is the second space after your period?!

Reply:
I am ashamed of myself and will go sit in the corner.